src/screens/SelectExerciseScreen.js

```diff
@@ -8,212 +9,120 @@ import {
+      "Let's build a work for" + this.state.selected.map(item => " " + item)
+    );
+    this.props.callback_setActiveView("exercisesScreen");
+  };
+
   render() {
     return (
-      <View>
+      <View style={{ height: "100%", width: "100%" }}>
         <Header>
-          <Left />
           <Body>
-            <Title>Select Workout</Title>
+            <Title>Select Muscle Groups</Title>
           </Body>
-          <Right />
         </Header>
-        <ScrollView
-          style={{
-            flex: 1,
-            width: "100%",
-            paddingLeft: 24,
-            paddingRight: 24
-          }}
-          contentContainerStyle={{
-            alignItems: "center"
-          }}
-        >
-          <View
-            style={{
-              flex: 1,
-              flexDirection: "row",
-              justifyContent: "space-between",
-              width: "100%"
-            }}
-          >
-            <Button
-              onPress={() => {
-                this.handleSelectedExerciseButtonClick(
-                  "exercisesScreen",
-                  "Arms"
-                );
-              }}
-              iconLeft
-              block
-              primary
-              bordered
-              style={{
-                height: 100,
-                width: "48%",
-                marginTop: 8,
-                marginBottom: 8,
-                borderColor: "#1C9963"
-              }}
-            >
-              <Icon name="hand" style={{ color: "#1C9963" }} />
-              <Text style={{ fontSize: 28, color: "#1C9963" }}>arms</Text>
-            </Button>
-
-            <Button
-              onPress={() => {
-                this.handleSelectedExerciseButtonClick(
-                  "exercisesScreen",
-                  "Back"
-                );
-              }}
-              iconLeft
-              block
-              bordered
-              primary
-              style={{
-                height: 100,
-                width: "48%",
-                marginTop: 8,
-                marginBottom: 8,
-                borderColor: "#1C9963"
-              }}
-            >
-              <Icon name="logo-android" style={{ color: "#1C9963" }} />
-              <Text style={{ fontSize: 28, color: "#1C9963" }}>back</Text>
-            </Button>
-          </View>
-
-          <View
-            style={{
-              flex: 1,
-              flexDirection: "row",
-              justifyContent: "space-between",
+        <View style={{ width: "100%" }}>
+          <ScrollView
+            contentContainerStyle={{
+              alignItems: "center",
+              paddingTop: 8,
+              paddingRight: 16,
+              paddingLeft: 16,
               width: "100%"
             }}
           >
+            <Text style={{ paddingTop: 8, paddingBottom: 8 }}>
+              Select which muscle groups you want to add to your workout!
+            </Text>
+            {this.renderSelectList(this.state.muscleGroups)}
```

Comment:
Any time I find myself creating a new `render` method– it's usually a good spot to break out into a separate component. (A `SelectList` component that takes in a `list` prop (`this.state.muscleGroups`))

Reply:
Thanks for the review :D 👍